docs/user/datastores/query_interceptor.rst

```diff
@@ -142,5 +147,59 @@ hours, or minutes. For example:
       }
     }
 
+Percentages can be defined in decimal form. e.g. .1 corresponds to 10%. Any query smaller than the first size will
+return 100% of the records. For example:
+
+.. code-block:: none
+
+    geomesa {
+      guard {
+        graduated {
+          "sftName" = [
+            { size = 1,  percentage = .8 }
+            { size = 10, percentage = .5  }
+            { percentage = .1 }
+          ]
+        }
+      }
+    }
+
+It is also possible to specify the attribute to use for the threading key in subsampling:
+
+.. code-block:: none
+
+    geomesa {
+      guard {
+        graduated {
+          "sftName" = [
+            { size = 1,  percentage = .8, attribute = "name" }
+            { size = 10, percentage = .5, attribute = "name"  }
+            { percentage = .1, attribute = "name" }
+          ]
+        }
+      }
+    }
+
+Additionally, it's possible to combine duration and percentage limits, with or without specifying an attribute:
+
+.. code-block:: none
+
+    geomesa {
+      guard {
+        graduated {
+          "sftName" = [
+            { size = 1,  duration = "60 days" }
+            { size = 10, duration = "3 days", percentage = .5  }
+            { duration = "1 day", percentage = .1, attribute = "name" }
+          ]
+        }
+      }
+    }
+
+In the above example, Any query with area less than 1 square degree will return all results and allow for any time
+range (given the time range doesn't trigger a full table scan). A query larger than 1 but less than 10 will block any
+query longer than 3 days AND only 50% of the results will be returned. Finally, any query larger than 10 must be less
+than 1 day in length AND only 10% of the results for any "name" will be returned.
```

Comment:
i find this behavior a little confusing but i think that's just how the graduated guard worked so no need to change it. thanks for the detailed explanation, it helps a lot!